commonTools/test/utilities/check-lfs-oids.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash    
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+target_branch=${1}
+source_branch=${2}
+
+for commit in $(git log --format=%H ${source_branch} --not ${target_branch})
```

Comment:
That is correct.  That should iterate over the all of the commits unique to the topic branch (source branch) not already in the target branch.